docker_setup_scripts/centos.sh

```diff
@@ -129,7 +130,11 @@ install_packages() {
   local toolset_package_suffixes=( "${TOOLSET_PACKAGE_SUFFIXES_COMMON[@]}" )
   if [[ $centos_major_version -eq 7 ]]; then
     toolset_prefix="devtoolset"
-    gcc_toolsets_to_install=( "${CENTOS7_GCC_TOOLSETS_TO_INSTALL[@]}" )
+    if [[ $( uname -m ) == "x86_64" ]]; then
+      gcc_toolsets_to_install=( "${CENTOS7_GCC_TOOLSETS_TO_INSTALL_X86_64[@]}" )
+    else
```

Comment:
Do we want to have `else if` here or is it always going to be aarch?

Reply:
Right now yes, we only support x86_64 and aarch64.